webpack.config.js

```diff
@@ -5,6 +5,11 @@ var definePlugin = new webpack.DefinePlugin({
     __DEBUG__: JSON.stringify(process.env.NODE_ENV === 'development'),
 });
 
+var providePlugin = new webpack.ProvidePlugin({
```

Comment:
I gave it a thought, and I actually think we shouldn't publicly polyfill *at all*, as we are not an app, but a module. So I think I'll use babel-runtime + babel-plugin-transform-runtime instead, or use polyfill and the ProvidePlugin (not sure which one is best yet).